src/Modules/OrchardCore.Commerce/Abstractions/IStripePaymentService.cs

```diff
@@ -8,13 +8,19 @@ namespace OrchardCore.Commerce.Abstractions;
 /// <summary>
 /// When implemented handles the payment and creates an order.
 /// </summary>
-public interface ICardPaymentService
+public interface IStripePaymentService
```

Comment:
This will break a lot of things. It was meant to be a generic Interface for any payment provider.

Reply:
This interface was already for Stripe only before this PR.

Reply:
In the OrchardCore.Commerce module you mean. Of course, there is only Stripe supported out of the box but nothing prevents me from using this Interface and creating my own. Now, renaming it means that I need to use the IStripePaymentService to create a IPaypalPaymentService which doesn't make any sense.

Reply:
Yes, you are right. But my point is that it wasn't supported even before this PR. Abstracting payment services and supporting other possible payment providers are not part of this PR and also not part of OCC yet.

Reply:
It has always been a target. That's why we used that Interface name.

Reply:
This is a Commerce abstraction project ... we talked about this in a meeting a long time ago. I don't see the issue and why we would want to only support Stripe ...

Reply:
The goal of an abstraction is to be abstract (name included).

Reply:
Also, about history, they moved the abstractions to different folders than what it was in the beginning. This is why it doesn't show any history for what it was in the beginning. You can't rely on Git history to say whether an Interface is new or not.

Reply:
This interface was [added for the Stripe support](https://github.com/OrchardCMS/OrchardCore.Commerce/commit/cb23b7c02be1502cbb8decdb55a97f01003f65b3) (it wasn't moved, this is the whole history of the file). Currently it's not possible to have anything else, until proper abstraction is added via #149. (this interface should be moved into a new OrchardCore.Commerce.PaymentProcessors.Stripe module during that issue) It is very Stripe-specific with the whole "payment intent" thing, and a more abstract interface will have to look differently. Because of that I'm certain there are no extensions this change could break.

Reply:
Ok then. You seem right. I don't see why this interface is required then. It is more confusing than anything and I understand why it is getting renamed now. 

You eventually probably need to make use of the IPaymentProvider and IPayment interfaces instead of ICardPaymentService and PaymentIntent then. Where a payment provider provides multiple ways to do payments.